src/helpers/index.ts

```diff
@@ -37,3 +44,29 @@ export const formatToArticleListItem = (
     sourceName: 'SPACEFORCE.mil',
   }
 }
+
+/** check if an object with publishedDate and status is Published */
```

Comment:
is this the logic you were considering moving to the cms repo? not sure if that has to be now or can be later. going to go ahead and give my ✅ since everything works as expected!